addons/web/static/lib/bootstrap/scss/_variables.scss

```diff
@@ -5,6 +5,9 @@
 
 // Color system
 
+$base-gray-color-hue: 210 !default;
+$base-gray-color-saturation: 0 !default;
```

Comment:
This is a lib file, you cannot modify it :grimacing: Especially since that feature has nothing to do with the web app, it is a website only feature. Handling colors is actually done in web_editor right now, this is where we set the gray palette, check for the `$o-gray-color-palette` variable. This is the map you should tweak, in our own files, not the bootstrap ones.